content/_es/help/verify-your-identity/troubleshoot-uploading-your-state-issued-id.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Solucionar problema al subir el documento de identidad emitido por el estado
+category: verify-your-identity
+order: 6
+---
+[Para la verificación de identidad](https://login.gov/es/help/verify-your-identity/how-to-add-images-of-your-state-issued-id/) se requiere una foto nítida de tu documento de identidad emitido por el estado. Si tomaste una foto que cumple con los requisitos, intenta los siguientes pasos si todavía recibes un error al subir tu identificación emitida por el estado.
+
+Tienes 10 intentos para subir tu documento. Una vez que realices los 10 intentos, el sistema te bloqueará durante seis horas como medida de seguridad. 
+
+Si los siguientes pasos no funcionan y no puedes completar el proceso de verificación de identidad, ponte en contacto con la agencia a la que intentas acceder.
+
+## Actualiza tu navegador  
+
+La causa del error podría ser un navegador obsoleto. Sigue las siguientes instrucciones para actualizar tu navegador: 
+
+* Google Chrome: <https://support.google.com/chrome/answer/95414?co=GENIE.Platform%3DDesktop&hl=en-GB>
+* Internet Explorer: <https://support.microsoft.com/en-us/help/17621/internet-explorer-downloads>
+* Mozilla Firefox: <https://support.mozilla.org/en-US/kb/update-firefox-latest-version>
+* Safari: <https://support.apple.com/en-us/HT204416>
```

Comment:
I notice some of these URLs have localized versions available, maybe we should bring them straight to the article in their preferred language?

```suggestion
* Google Chrome: <https://support.google.com/chrome/answer/95414?co=GENIE.Platform%3DDesktop&hl=es-419>
* Internet Explorer: <https://support.microsoft.com/es-es/windows/descargas-de-internet-explorer-d49e1f0d-571c-9a7b-d97e-be248806ca70>
* Mozilla Firefox: <https://support.mozilla.org/es/kb/actualizar-firefox-la-ultima-version>
* Safari: <https://support.apple.com/es-lamr/HT204416>
```

Same applies to other help articles listings.